README.md

```diff
@@ -324,6 +322,16 @@ We send metrics for Jobs in the following states:
   autoscaler, since it represents work that will start soon.
 - **Running**: an agent is actively executing this job.
 
+Detailed explanations about some of the metrics: 
+
+**RunningJobsCount**: This metric counts the number of jobs that are currently in the “running” state. These jobs have been picked up by agents and are actively being executed.
+
+**UnfinishedJobsCount**: This metric includes all jobs that have been scheduled but have not yet finished. This count would include the jobs that are in the “running,” “scheduled”.
+
+**ScheduledJobsCount**: This metric refers to jobs that have been scheduled but not yet started by any agent. These jobs are in the queue, waiting for an available agent to start executing them.
+
+**WaitingJobsCount**: This metric counts jobs that are in a “waiting” state, which could mean they are waiting on dependencies to resolve, on other jobs to finish, or on any other condition that needs to be met before they can be scheduled. When a queue is specified, only that queue's metrics are published.
```

Comment:
Ah, the sentence above wasn't deleted, it somehow ended up here. Move it back? where it was?

```suggestion
**WaitingJobsCount**: This metric counts jobs that are in a “waiting” state, which could mean they are waiting on dependencies to resolve, on other jobs to finish, or on any other condition that needs to be met before they can be scheduled.
```